content/docs/apps/continuous-deployment.md

```diff
@@ -28,6 +28,34 @@ cloud.gov does not yet provide a CI/CD (continuous integration/continuous deploy
 
 You can configure your code repositories, [spaces]({{< relref "docs/getting-started/concepts.md#spaces" >}}), and CI/CD service together to enable automated or semi-automated deployments to your environments (such as development, staging, and production environments). For deployments in each environment, you can configure access control and testing requirements according to your project's needs.
 
+An overview of a common workflow for CI/CD with cloud.gov:
```

Comment:
This may be a common workflow, but it's not a recommended one because you have long-lived distinct branches which introduces a lot of integration problems. 

"A key correlation with high performing orgs is trunk-based delivery"

Reply:
I'm not sure we want to get into specific recommendations about branching workflows. Maybe we can simplify this to something like commit -> test -> deploy, without reference to branches. Although that might be simple enough that we don't need a diagram at all.

Reply:
I'd be happy not getting into specific recommendations for branching workflows, if we can avoid it. The question I'm attempting to answer is people asking "so, how would I set up staging and production environments in cloud.gov, using CI/CD?" - is there a better way to answer that without getting into the details? Diagram is optional, but diagrams can help a lot.

Reply:
In the end, lets leave the diagram exactly as is, but note that: "This workflow only demonstrates how the components of CICD and cloud.gov could work together, and isn't a recommendation. See (More resources) below"

Then I'm on the hook for providing some links(!).